agr_java_core/src/main/java/org/alliancegenome/es/index/site/schema/settings/VariantIndexSettings.java

```diff
@@ -26,11 +26,11 @@ public void buildSettings() throws IOException {
                 //.field("merge.scheduler.max_thread_count", "10")
                 //.field("merge.policy.floor_segment", "200mb")
                 //.field("merge.policy.max_merged_segment", "500mb")
-                .field("translog.sync_interval", "15s")
+                .field("translog.sync_interval", "60s")
                 .field("translog.flush_threshold_size", "4096mb")
                 .field("translog.durability", "async");
                 //.field("codec", "best_compression");
-            buildAnalysis(true);
+            //buildAnalysis(true);
```

Comment:
@jt15 we may need to bring back the `buildAnalysis` for the variant indexes but for now going to try without it.